### PR TITLE
Website refactoring

### PR DESCRIPTION
**Pull request by skeydan**
- add new section "get started"
- move low-level intro to "get technical"
- add "resources" tab

### CODE REVIEW

File: content/start/guess_the_correlation/index.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,632 @@
...
+```{r}
+length(train_dl)
+```
+
+    [1] 157
+
+To access the first batch, we create an iterator from the `DataLoader` and ask it for the first batch. Even if it weren't for plotting, you might do this just to check that the dimensions look ok:
+
+```{r}
+iter <- train_dl$.iter()
```
Comment (dfalbel) [inline content/start/guess_the_correlation/index.Rmarkdown:224]: can we use `dataloader_make_iter` and `dataloader_next` here?

File: content/start/what_if/index.Rmarkdown [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+      self$fc1() %>%
+      nnf_relu() %>%
+      self$bn4() %>%
+      
+      self$fc2()
+  }
+)
+```
+
+Once you're found that a regularizing measure works -- meaning, performance on the validation set is similar to that on the training set, or maybe even better -- you can go back and add more capacity to the network: add more layers, train for a longer time, etc. Maybe you'll arrive at better performance overall!
```
Comment (dfalbel) [inline content/start/what_if/index.Rmarkdown:320]: I think it would be nice to have a prediction example at the end showing the plot and the correlation predicted by the model.

File: content/start/what_if/index.markdown [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+      self$fc1() %>%
+      nnf_relu() %>%
+      self$bn4() %>%
+      
+      self$fc2()
+  }
+)
+```
+
+Once you're found that a regularizing measure works -- meaning, performance on the validation set is similar to that on the training set, or maybe even better -- you can go back and add more capacity to the network: add more layers, train for a longer time, etc. Maybe you'll arrive at better performance overall!
```
Comment (dfalbel) [inline content/start/what_if/index.markdown:321]: Maybe we could have a 'what if I want to train on the GPU'?

File: content/resources/index.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+title: resources
```
Comment (dfalbel) [inline content/resources/index.md:2]: Should this be in title case?

**Review by dfalbel — COMMENTED** [review]
Hey @skeydan ! I loved the guess the correlation tutorial! Also It's really nice how you added the many possible variations in the 'what if' guide! 

I think this make torch more easier to use get started!!!

I added a few comments, let me know what you think!

File: content/start/guess_the_correlation/index.Rmarkdown (hunk 1 above)
Comment (skeydan) [reply content/start/guess_the_correlation/index.Rmarkdown:224]: oh yeah, for sure, thank you!

File: content/start/what_if/index.markdown (hunk 3 above)
Comment (skeydan) [reply content/start/what_if/index.markdown:321]: yes, you're right, I'll add that! It'll be a frequent question

File: content/resources/index.md (hunk 4 above)
Comment (skeydan) [reply content/resources/index.md:2]: hm no, well in fact i wanted them all to be lower case (looks more cool :-))

but i just now noticed that "Welcome" is upper case ... lowercased that one then :-)

File: content/start/what_if/index.Rmarkdown (hunk 2 above)
Comment (skeydan) [reply content/start/what_if/index.Rmarkdown:320]: hm, do you mean in the what-if section? Or perhaps just at the end of the main (first) tutorial?

Comment (dfalbel) [reply content/start/what_if/index.Rmarkdown:320]: I placed the comment in the wrong place! Yeah, in the end of the first tutorial :)

Comment (skeydan) [reply content/start/what_if/index.Rmarkdown:320]: will do !

**Comment by skeydan** [comment]
hi @dfalbel, added the performance plot and the GPU section, these were great suggestions!

File: content/start/guess_the_correlation/index.markdown [hunk 5]
```diff
@@ -626,21 +625,32 @@ for (epoch in 1:num_epochs) {
...
 
 ```r
 model$eval()
 
 test_batch <- function(b) {
-
-  output <- model(b$x)
-  loss <- nnf_mse_loss(output, b$y$unsqueeze(2))
+  
+  output <- model(b$x$to(device = "cuda"))
```
Comment (dfalbel) [inline content/start/guess_the_correlation/index.markdown:642]: I think we don't need to move to cuda here right?
